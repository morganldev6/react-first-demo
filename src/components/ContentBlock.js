import React from "react";

const ContentBlock = ({ title, content }) => {
  return (
    <div className="contentBlock">
      <h2>{title}:</h2>
      <p>{content}</p>
    </div>
  );
};

export default ContentBlock;
