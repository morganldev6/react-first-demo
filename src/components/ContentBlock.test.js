import React from "react";
import { render, cleanup } from "@testing-library/react";

import ContentBlock from "./ContentBlock";

afterEach(cleanup);

describe("ContentBlock shows each line item describing the movie", () => {
  test("Renders correctly", () => {
    const { container } = render(<ContentBlock />);
    expect(container).toMatchSnapshot();
  });
});
