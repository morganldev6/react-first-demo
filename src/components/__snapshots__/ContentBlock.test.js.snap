// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentBlock shows each line item describing the movie Renders correctly 1`] = `
<div>
  <div
    class="contentBlock"
  >
    <h2>
      :
    </h2>
    <p />
  </div>
</div>
`;
